### DISCUSSION TITLE
Center link on element

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1876] [Q&A]
Hello,
I would like my links to be connected to the centre of my elements. I can do this but as soon as I add a title outside the element, the link moves and is no longer in the centre of the element.

Here is an example with and without text on my element:

<img width="224" alt="Capture d’écran 2022-11-03 145915" src="https://user-images.githubusercontent.com/83480447/199740797-a93e4eaa-8bc8-4823-98ac-ee6e498e79f9.png">
<img width="187" alt="Capture d’écran 2022-11-03 145939" src="https://user-images.githubusercontent.com/83480447/199740795-5208d0ca-7979-4654-bc8f-28342f45cb27.png">
<img width="176" alt="Capture d’écran 2022-11-03 145953" src="https://user-images.githubusercontent.com/83480447/199740789-24223351-5275-4f03-8f95-9791967b435f.png">

You see when I selected the element its select the title too. So, the link isn't in the center of the square grid.

Thank you in advance for your feedback.

**Comment by kumilingus** [comment]
Duplicate of https://github.com/clientIO/joint/discussions/1860.
